src/api/app/views/webui/attribute/_issue_fields.html.haml

```diff
@@ -0,0 +1,6 @@
+.nested-fields
+  .field
+    = f.select :issue_tracker_id, IssueTracker.all.collect { |tracker| [ tracker.name, tracker.id ] }.sort, include_blank: false
```

Comment:
If you time it would be nice to move this code to a the controllers but I don't mind if you prefer to do that later either:
```ruby
IssueTracker.all.collect { |tracker| [ tracker.name, tracker.id ] }.sort
```